website/docs/r/workitemquery_permissions.html.markdown

```diff
@@ -0,0 +1,130 @@
+
+## Permission levels
+
+Permission for Work Item Queries within Azure DevOps can be applied on two different levels.
+Those levels are reflected by specifying (or omitting) values for the arguments `project_id` and `path`.
+
+### Project level
+
+Permissions for all Work Item Queries inside a project (existing or newly created ones) are specified, if only the argument `project_id` has a value.
+
+#### Example usage
+
+```hcl
+resource "azuredevops_workitemquery_permissions" "project-wiq-root-permissions" {
+  project_id  = azuredevops_project.project.id  
+  principal   = data.azuredevops_group.project-readers.id
+  permissions = {
+    CreateRepository = "Deny"
+    DeleteRepository = "Deny"
+    RenameRepository = "NotSet"
+  }
+}
+```
+
+### Shared Queries folder level
+
+Permissions for a specific folder inside Shared Queries are specified if the arguments `project_id` and `path` are set.
+
+~> **Note** To set permissions for the Shared Queries folder itself use `/` as path value
+
+#### Example usage
+
+```hcl
+resource "azuredevops_workitemquery_permissions" "wiq-folder-permissions" {
+  project_id = azuredevops_project.project.id
+  path = "/Team"
+  principal   = data.azuredevops_group.project-readers.id
+  permissions = {
+    Contribute = "Allow"
+    Delete = "Deny"
+    Read = "NotSet"
+  }
+}
+```
+
+## Example Usage
+
+```hcl
+resource "azuredevops_project" "project" {
+  project_name       = "Test Project"
+  description        = "Test Project Description"
+  visibility         = "private"
+  version_control    = "Git"
+  work_item_template = "Agile"
+}
+
+data "azuredevops_group" "project-readers" {
+  project_id = azuredevops_project.project.id
+  name       = "Readers"
+}
+
+data "azuredevops_group" "project-contributors" {
+  project_id = azuredevops_project.project.id
+  name       = "Contributors"
+}
+
+resource "azuredevops_workitemquery_permissions" "wiq-project-permissions" {
+  project_id  = azuredevops_project.project.id
+  principal   = data.azuredevops_group.project-readers.id
+  permissions = {
+    Read              = "Allow"
+    Delete            = "Deny"
+    Contribute        = "Deny"
+    ManagePermissions = "Deny"
+  }
+}
+
+resource "azuredevops_workitemquery_permissions" "wiq-sharedqueries-permissions" {
+  project_id = azuredevops_project.project.id
+  path = "/"
+  principal   = data.azuredevops_group.project-contributors.id
+  permissions = {
+    FullControl              = "Allow"
+  }
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project_id` - (Required) The ID of the project to assign the permissions.
+* `path` - (Optional) Path to a query or folder beneath `Shared Queries`
+* `principal` - (Required) The **group** principal to assign the permissions.
+* `replace` - (Optional) Replace (`true`) or merge (`false`) the permissions. Default: `true`
+* `permissions` - (Required) the permissions to assign. The follwing permissions are available
```

Comment:
```suggestion
* `permissions` - (Required) the permissions to assign. The following permissions are available.
```

Reply:
I  thought `misspell`would detect that. But it doesn't. Seems to me that this tool isn't a great help anyway. Any better alternative?

Reply:
`misspell` is outdated and no one is maintain that repo. Didn't find a better one. :(

Reply:
I gave [markdown-spellcheck](https://www.npmjs.com/package/markdown-spellcheck) a try.  Output is well readable. But I doubt we'll get it to work in some automatic way, because, like with the documentation about permissions, we'll have numerous "special" words what will cause false positives.

![image](https://user-images.githubusercontent.com/14177833/88906913-4c765580-d258-11ea-9e19-59058319d381.png)

Reply:
I can test if we can integrate this tool to CI pipeline.